h/static/scripts/filter/test/document-title-test.js

```diff
@@ -1,75 +0,0 @@
-'use strict';
-
-var documentTitle = require('../document-title');
-
-describe('documentTitle', function() {
-
-  it('returns the title linked if the document has title and uri', function() {
```

Comment:
The revised tests look clear to me. If we had a lot more cases then using a data-driven test is a useful way to make things more concise.